web-api/src/storage.rs

```diff
@@ -104,7 +104,7 @@ pub(crate) trait Document {
     ) -> Result<(), DeletionError>;
 }
 
-#[async_trait]
+#[async_trait(?Send)]
```

Comment:
Can't we do it only on the trait where we need it?

Reply:
Did we start adding this due to adding more generics in async-trait traits?

Reply:
> Did we start adding this due to adding more generics in async-trait traits?

yes, the `Send` bounds were in conflict with some of the iterator arguments. since we can't enforce the `Send` bounds lint because of `actix-web`, we decided to relax it here as well.

> Can't we do it only on the trait where we need it?

i'll double check and only relax the bounds where needed.